app/src/main/java/com/android/sample/ui/components/PlusButtonToCreate.kt

```diff
@@ -20,7 +20,7 @@ import com.android.sample.ui.navigation.NavigationActions
 import com.android.sample.ui.navigation.TopLevelDestinations
 
 @Composable
-fun PlusButtonToCreate(navigationActions: NavigationActions) {
+fun PlusButtonToCreate(navigationActions: NavigationActions, category: String) {
```

Comment:
I suggest renaiming this function since it does not only handle cases where the plus button navigates to create activity